GOVERNANCE.md

```diff
@@ -21,16 +21,17 @@ future addition of committee seats), the Envoy core proxy maintainer seat will b
 tie with a super vote.
 
 ## Steering Committee
-The steering committee will be a 5-member body that will make directional and any strategic
+The steering committee will be a 6-member body that will make directional and any strategic
```

Comment:
IMO I don't think the SC should be capped at a certain number of members. Therefore, I suggest we remove the number of members from this sentence and focus on what the SC is all about.

@richarddli @mattklein123 feel free to chime in if you disagree.

Reply:
That's fine with me.

Reply:
@Xunzhuo I suggest:
/The steering committee will be a 6-member body that will make directional and any strategic technical decisions on the Envoy gateway project./The steering committee is a governing body that makes directional and strategic decisions for the Envoy gateway project./

Reply:
I would vote to keep it at 6. I know in practice it's not an issue but I also think that having an unbounded committee is not a recipe for good governance.

Reply:
@richarddli good point. I'm fine keeping it at 6.

@Xunzhuo please update as follows:

/The steering committee will be a 6-member body that will make directional and any strategic technical decisions on the Envoy gateway project./The steering committee is a 6-member governing body that makes directional and strategic decisions for the Envoy gateway project./

Reply:
Thanks, I will update it later.